src/arithmetic/list_funcs/list_funcs.c

```diff
@@ -365,6 +458,30 @@ void Register_ListFuncs() {
 	func_desc = AR_FuncDescNew("tolist", AR_TOLIST, 0, VAR_ARG_LEN, types, ret_type, true, true);
 	AR_RegFunc(func_desc);
 
+	types = array_new(SIType, 1);
+	array_append(types, T_ARRAY | T_NULL);
+	ret_type = T_ARRAY|  T_NULL;
```

Comment:
`T_ARRAY | T_NULL;`

Reply:
Ok. Fixed in [ae328c9](https://github.com/RedisGraph/RedisGraph/pull/2605/commits/ae328c970c5491deac01358f985fa6aab2cd6ff7)